main.tf

```diff
@@ -1,31 +1,53 @@
-provider "aws" {
-  region = "eu-west-3"
-}
-
-resource "aws_vpc" "test-vpc" {
+//TODO: rename VPC
+resource "aws_vpc" "nginx_vpc" {
   cidr_block = "10.0.0.0/16"
-
+  
   tags = {
-    Name = "Premium Auto VPC"
+    Name = "Nginx VPC"
   }  
 }
 
 //Application Load Balancer
-resource "aws_lb" "alb" {
-  name = "premium-auto-lb"
+//TODO add listeners and target groups
+
+resource "aws_lb" "app_lb" {
+  name = "nginx-load-balancer"
   load_balancer_type = "application"
-  security_groups = [aws_security_group.sg-lb.id]
+  security_groups = [aws_security_group.sg_lb.id]
   subnets = aws_subnet.public_subnet.*.id
   enable_cross_zone_load_balancing = true
+  
+}
+
+resource "aws_lb_target_group" "group" {
+  name = "terraform-alb-target"
```

Comment:
Rename to `nginx-alb-target`

Reply:
Renamed terraform-alb-target to nginx-alb-taget